core/src/main/scala/kafka/coordinator/group/GroupMetadata.scala

```diff
@@ -289,6 +290,35 @@ private[group] class GroupMetadata(val groupId: String, initialState: GroupState
     pendingOffsetCommits ++= offsets
   }
 
+  def prepareTxnOffsetCommit(producerId: Long, offsets: mutable.Map[TopicPartition, OffsetAndMetadata]) {
+    val producerOffsets = pendingTransactionalOffsetCommits.getOrElseUpdate(producerId, mutable.Map.empty[TopicPartition, OffsetAndMetadata])
```

Comment:
This is the existing behavior for non transactional offset commits as well, right?